### PR TITLE
Hydrological Processes

### PR DESCRIPTION
**Pull request by aditibhaskar**

### CODE REVIEW

**Comment by aditibhaskar** [comment]
New style for Hydrological Processes

File: hydrological-processes.csl [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291544-8452/homepage/ForAuthors.html" rel="documentation"/>
+    <author>
+      <name>Aditi Bhaskar</name>
+      <email>abhaskar@usgs.gov</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="geography"/>
+    <issn>0885-6087</issn>
+    <issn>1099-1085</issn>
+    <eissn>1099-1085</eissn>
```
Comment (zuphilip) [inline hydrological-processes.csl]: Delete one of these lines with the same issn.

Comment (zuphilip) [inline hydrological-processes.csl]: Preferable, the `eissn` should stay, that seems to be correct.

Comment (aditibhaskar) [inline hydrological-processes.csl]: I see two ISSNs, one for print and one online.  So is the first one the
main ISSN and the second one for eISSN?

*Print ISSN:* 0885-6087
*Online ISSN:* 1099-1085

http://authorservices.wiley.com/bauthor/Aims_scope.asp?ref=0885-6087&site=1

On Tue, Aug 4, 2015 at 2:36 PM, Philipp Zumstein <notifications@github.com>
wrote:

> In hydrological-processes.csl
> <https://github.com/citation-style-language/styles/pull/1643#discussion_r36225126>
> :
>
> > +
> > +  <info>
> > +    <title>Hydrological Processes</title>
> > +    <id>http://www.zotero.org/styles/hydrological-processes</id>
> > +    <link href="http://www.zotero.org/styles/hydrological-processes" rel="self"/>
> > +    <link href="http://www.zotero.org/styles/aquatic-conservation" rel="template"/>
> > +    <link href="http://onlinelibrary.wiley.com/journal/10.1002/%28ISSN%291544-8452/homepage/ForAuthors.html" rel="documentation"/>
> > +    <author>
> > +      <name>Aditi Bhaskar</name>
> > +      <email>abhaskar@usgs.gov</email>
> > +    </author>
> > +    <category citation-format="author-date"/>
> > +    <category field="geography"/>
> > +    <issn>0885-6087</issn>
> > +    <issn>1099-1085</issn>
> > +    <eissn>1099-1085</eissn>
>
> Preferable, the eissn should stay, that seems to be correct.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1643/files#r36225126>
> .
>



-- 
Aditi S. Bhaskar
NSF Postdoctoral Research Fellow
U.S. Geological Survey, Eastern Geographic Science Center
12201 Sunrise Valley Drive, MSN 521, Reston, VA 20192
Phone: (703) 648-5157
http://aditibhaskar.weebly.com/

Comment (zuphilip) [inline hydrological-processes.csl]: We would want
```
    <issn>0885-6087</issn>
    <eissn>1099-1085</eissn>
```

Comment (rmzelle) [inline hydrological-processes.csl]: Yes. (eISSN stand for "electronic International Standard Serial Number")

You can update your pull request via the pencil (edit) button at https://github.com/aditibhaskar/styles/blob/patch-1/hydrological-processes.csl

**Comment by rmzelle** [comment]
Can you update the "documentation" link to http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1099-1085/homepage/ForAuthors.html as well? It still point to the journal *Population, Space and Place*.

**Comment by aditibhaskar** [comment]
Yes, done

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**